src/shape.cc

```diff
@@ -6,8 +6,8 @@
 
 namespace nigiri {
 
-shapes_storage_t create_shapes_storage(std::filesystem::path const& path) {
-  constexpr auto const kMode = cista::mmap::protection::WRITE;
+shapes_storage_t create_shapes_storage(std::filesystem::path const& path,
+                                       cista::mmap::protection const kMode) {
   return {
       cista::basic_mmap_vec<geo::latlng, std::uint64_t>{cista::mmap{
           fmt::format("{}_data.bin", path.generic_string()).c_str(), kMode}},
```

Comment:
```suggestion
shapes_storage_t create_shapes_storage(std::filesystem::path const& path,
                                       cista::mmap::protection const mode) {
  return {
      cista::basic_mmap_vec<geo::latlng, std::uint64_t>{cista::mmap{
          fmt::format("{}_data.bin", path.generic_string()).c_str(), mode}},
```

Reply:
`kCamelCase` is a compile time constant

Reply:
Thank you for explaining. I didn't know about this convention.